addons/web/static/src/js/views/search/search_inputs.js

```diff
@@ -346,7 +346,8 @@ var BooleanField = SelectionField.extend(/** @lends instance.web.search.BooleanF
  */
 var DateField = Field.extend(/** @lends instance.web.search.DateField# */{
     value_from: function (facetValue) {
-        return time.date_to_str(facetValue.get('value'));
+        var serverFormat = (this.attrs && this.attrs.type === 'datetime') ? 'YYYY-MM-DD HH:mm:ss' : 'YYYY-MM-DD';
```

Comment:
it's weird to check if it is a datetime in the DateField widget, whereas there is a specific DateTimeField one

Reply:
@aab-odoo 
Yes it is
I can do the choice of the format of date/time in the init(), either way that check would have to be done in the super Class DateField